functors_task_2/student/include/impl.h

```diff
@@ -65,7 +65,17 @@ class DataBrowser
         // find reader
         // check for errors
         // call functor
-        return false;
+        const auto& it = m_dataReaders.find(userId);
```

Comment:
Please start each new task from the master branch, don't include this into your next prs!